components/chart/linearTimeSeries.js

```diff
@@ -0,0 +1,127 @@
+(function(d3, fc) {
+    'use strict';
+
+    fc.charts.linearTimeSeries = function() {
+
+        var xScale = d3.time.scale();
```

Comment:
should this be out own time scale - that permits discontinuities? you'd just have to rebind `discontinuityProvider` to allow people to use it.